docs/manual/java/releases/Migration15.md

```diff
@@ -114,12 +114,20 @@ my-database {
 }
 ```
 
-
-
 ## Service Discovery
 
 When opting in to Akka Cluster Bootstrapping as a mechanism for Cluster formation you will have to setup a [[Service Discovery|Cluster#Akka-Discovery]]  method for nodes to locate each other.
 
+## Production Settings
+
+New defaults have been added to the Lagom clustering configuration. 
+
+The first default configuration concerns how long a node should try to join an cluster. This is configured by the setting `akka.cluster.shutdown-after-unsuccessful-join-seed-nodes`. This is an existing Akka Cluster setting that is by default set to `off`, but Lagom will redefine it to 60 seconds. After that period, the Actor System will shutdown if it fails to join a cluster.
+
+The second important change is the default value for `lagom.cluster.exit-jvm-when-system-terminated`. This was previously `off`, but we always recommended it to be `on`  in production environments. As of Lagom 1.5.0, that setting defaults to `on`. When enabled, Lagom will exit the JVM when the application leave the cluster or fail to join the cluster. In Dev and Test mode, this setting is automatically set to `off`.
```

Comment:
There is an extra space after `on` in `always recommended it to be `on`  in production`

Reply:
Hi @aldenml, thanks for the reviews!

Quick tip: github supports making suggestions. Use the icon:

![Screen Shot 2019-03-22 at 09 32 43](https://user-images.githubusercontent.com/762126/54810093-793e5400-4c85-11e9-8aa8-fe67c52966dd.png)

and make the edit. Suggestions are limited to one-line changes but are very helpful and speed up the process a lot.

Reply:
thanks for the tip!